weather-schema.json

```diff
@@ -54,23 +54,30 @@
               "type": "string"
             },
             "visibility": {
-              "type": "string",
-              "enum": [
-                "veryPoor",
-                "poor",
-                "moderate",
-                "good",
-                "veryGood",
-                "excellent"
+              "anyOf": [
+                {
+                  "type": "string",
+                  "enum": [
+                    "veryPoor",
+                    "poor",
+                    "moderate",
+                    "good",
+                    "veryGood",
+                    "excellent"
+                  ]
+                },
+                {
+                  "type": "number"
```

Comment:
can we define boundaries for number?

Reply:
maybe minimum can be set to 0

Reply:
Fixed in 1c54ff0accf4d942a35c9835b8236432e309a0cf